web/Views/Admin/TenantList.cshtml

```diff
@@ -38,4 +52,11 @@
         </tr>
     }
     </tbody>
-</table>
\ No newline at end of file
+</table>
+
+@using (Html.BeginForm("ImportTenants", "Admin", FormMethod.Post, new { enctype = "multipart/form-data" }))
+{
+    <div>Please upload a CSV file containing tenant information</div>

```

Comment:
Can you update this to make it clear that it will replace all existing tenant records?